server/server.py

```diff
@@ -45,6 +47,8 @@
 
 try:
     app = Flask(__name__)
+    talisman = Talisman(app)
+    csrf = SeaSurf(app)
```

Comment:
some brief inline docs here might be nice because neither of these library names are self explanatory 😅

Reply:
aha ok i'll add that 😅